File: output/CodeGen/avatar/Python/Java_source/codeforces_265_B.java
public class codeforces_265_B {
    public int findLengthOfLastWord(String s) {
        int ans = 0;
        int last = 0;
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) =='') {
                last = i;
            }
            if (s.charAt(i)!='') {
                ans += i - last;
                last = i;
            }
        }
        return ans + 1;
    }
}

File: output/ChatGPT/avatar/Python/C++/codeforces_265_B.cpp
#include <iostream>
#include <vector>
#include <cmath>

int main() {
    int n;
    std::cin >> n;
    std::vector<int> l;
    for (int i = 0; i < n; i++) {
        int num;
        std::cin >> num;
        l.push_back(num);
    }
    int ans = n - 1;
    int last = 0;
    for (int i : l) {
        ans += std::abs(last - i) + 1;
        last = i;
    }
    std::cout << ans << std::endl;

    return 0;
}